src/pretix/control/views/organizer.py

```diff
@@ -1263,6 +1265,26 @@ def webhook(self):
     def get_queryset(self):
         return self.webhook.calls.order_by('-datetime')
 
+    def post(self, request, *args, **kwargs):
+        if request.POST.get("action") == "expedite":
+            self.request.organizer.log_action('pretix.webhook.retries.expedited', user=self.request.user, data={
+                'webhook': self.webhook.pk,
+            })
+            manually_retry_all_calls.apply_async(args=(self.webhook.pk,))
+            messages.success(request, _('All requests will now be scheduled for an immediate attempt. Please '
+                                        'allow for a few minutes before they are processed.'))
+        elif request.POST.get("action") == "drop":
+            self.request.organizer.log_action('pretix.webhook.retries.dropped', user=self.request.user, data={
+                'webhook': self.webhook.pk,
+            })
+            self.webhook.retries.all().delete()
+            messages.success(request, _('All requests will now be scheduled for an immediate attempt. Please '
+                                        'allow for a few minutes before they are processed.'))
```

Comment:
The message needs to be about deleted retries. Something like:

```suggestion
            messages.success(request, _('All unprocessed webhooks have been stopped from retrying.'))
```